x-pack/plugins/observability/public/hooks/use_fetch_rule.ts

```diff
@@ -61,7 +61,7 @@ export function useFetchRule({ ruleId }: { ruleId?: string }): UseFetchRuleRespo
       onError: (error: Error) => {
         toasts.addError(error, {
           title: i18n.translate('xpack.observability.ruleDetails.ruleLoadError', {
-            defaultMessage: 'Unable to load rule. Reason: {message}',
+            defaultMessage: 'Unable to load rule',
```

Comment:
Remove the message from the title as it's a duplication and it appears in the toast body.